include/clink/feature/one_hot_encoder.h

```diff
@@ -65,6 +76,9 @@ class OneHotEncoderModel {
 
   // Model data of OneHotEncoderModel.
   OneHotEncoderModelDataProto model_data_;
+
+  tfrt::HostContext *host_;
```

Comment:
Do we need to keep this field as member variable?

Reply:
Yeah it is usually a good idea to learn/from the existing practice. On the other hand we still need to know why we need this variable -- otherwise we might have followed a wrong convention :)

Note that `FilterDataset` actually uses `host_` in `filter_dataset.cc`. It appears that every subclass of `Dataset` in TFRT needs `host_` to create iteration.

If we don't have any good reason to keep `host_ ` as member variable here, it seems better to remove this for now.

Reply:
OK. I'll make the change.